lib/rest-api.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Internationalization-related functions for the Gutenberg editor plugin.
```

Comment:
Inaccurate description. Probably something about these being enhancements to existing endpoints? A bit unclear where the line should be drawn between `compat.php` and this one.

Reply:
Lol woops, copy pasta.

Reply:
This should probably go into `compat.php`. Didn't realize what that was until now.